web/routes.go

```diff
@@ -39,6 +40,12 @@ var routes = []route{
 		"/health",
 		healthHandler,
 	},
+	route{
+		"job_status",
+		"GET",
+		"/job_status",
```

Comment:
I think we need to add something like `/{id}` to the end of the route to get this working.